src/fediblockhole/__init__.py

```diff
@@ -787,6 +795,7 @@ def setup_argparse():
     ap.add_argument('-b', '--block-audit-file', dest="blocklist_auditfile", help="Save blocklist auditfile to this location.")
     ap.add_argument('--merge-threshold', type=int, help="Merge threshold value")
     ap.add_argument('--merge-threshold-type', choices=['count', 'pct'], help="Type of merge threshold to use.")
+    ap.add_argument('--override-private-comment', dest='override_private_comment', help="Enforces a private comment for all blocks.")
```

Comment:
The help here could perhaps be a little clearer about what it does?

How about `Override private_comment with this string for new blocks when pushing blocklists`? 

Have I properly understood the purpose of the feature?

Reply:
Yes, that's correct.